deployment/conf/supported_apps_w_extensions.json

```diff
@@ -0,0 +1,498 @@
+{
+  "apps": {
+    "SRA Reads": {
+      "title": "SRA Reads",
+      "app": "kb_uploadmethods/import_fastq_sra_as_reads_from_staging",
+      "output_type": [
+        "KBaseFile.SingleEndLibrary",
+        "KBaseFile.PairedEndLibrary"
+      ],
+      "extensions": [
+        "sra"
+      ],
+      "id": 0
```

Comment:
I'd move the ID so it's the first item in the object.

I'd also argue for using a text ID, because it is very easy to make mistakes when dealing with numbers -- there's nothing intuitive that links SRA reads to `0` or FastQ reads to `1` or `2`, whereas if you make the IDs `sra_reads` and `fastq_reads_interleaved`, it's a lot clearer which ID corresponds to which type. There's less likelihood for error in the "types" section of the file, and it can also be much more easily read and understood by anyone looking at the file.

Reply:
This file is autogenerated.
I guess that is a good point, if this is a json file, it should be human readable? It would just add an extra step to convert it back to integers in the api response

Reply:
@ialarmedalien I could make it return this instead



```
"tsv": [
      {
        "id": 9,
        "title": "media",
        "app_weight": 1
      },
      {
        "id": 10,
        "title": "expression_matrix",
        "app_weight": 1
      },
      {
        "id": 11,
        "title": "metabolic_annotations",
        "app_weight": 1
      },
      {
        "id": 12,
        "title": "bulk_metabolic_annotations",
        "app_weight": 1
      },
      {
        "id": 13,
        "title": "fba_model",
        "app_weight": 1
      },
      {
        "id": 14,
        "title": "phenotype_set",
        "app_weight": 1
      }
    ],
```